src/tests/ftest/soak/test_base.py

```diff
@@ -13,19 +13,20 @@
 import random
 from filecmp import cmp
 from apricot import TestWithServers
-from general_utils import run_command, DaosTestError, get_log_file
+from general_utils import run_command, DaosTestError
 from command_utils_base import CommandFailure
 import slurm_utils
 from ClusterShell.NodeSet import NodeSet
 from getpass import getuser
 import socket
 from agent_utils import include_local_host
-from utils import DDHHMMSS_format, add_pools, get_remote_logs, \
+from utils import DDHHMMSS_format, add_pools, get_remote_dir, \
```

Comment:
(pylint-syntax-error) Cannot import 'utils' due to syntax error 'EOL while scanning string literal (<unknown>, line 824)'